_data/software.yml

```diff
@@ -82,6 +73,15 @@ core:
       anchor: 5gc
       software:
 
+        - title: Open5GS
+          contributors: Open5GS
+          license: GNU AGPLv3
+          status: Active
+          community: Community support available
+          url: https://open5gs.org
+          desc: |
+            Open5GS offers an open source implementations of Release 16 a 4G/5G NSA compliant EPC, and a 5G SA core. The 4G/5G NSA network components and 5G SA network functions are written in C and  distributed under  the  AGPLv3  license. It  is  compatible  with  a variety of Linux distributions, such as Debian, Ubuntu, Fedora, and  CentOS, as well as FreeBSD and macOS. Differently from  other EPCs, Open5GS supports the delivery of voice calls and text messages through the LTE network instead of relying on traditional circuit switching networks. This is achieved by leveraging third party Voice over LTE (VoLTE) and SG-SMS (SMSoSGs) solutions, respectively; such as those from Kamailio and Osmocom. Open5GS includes PCRF/PCF modules, through which operators can specify network policies in real-time, including prioritizing a certain type of traffic. Advanced 5G SA core functionalities, such as slicing, are currently under development. Open5GS is being used with a variety of 4G and 5G software stacks and physical radio equipment. A list of tested compatible equipment is available at https://open5gs.org/open5gs/docs/hardware/01-genodebs
```

Comment:
Could you either remove "an" or use "implementation"?

Reply:
Ah, good catch!